src/config/rds_production_dependency.go

```diff
@@ -41,5 +44,7 @@ func (r *rdsProductionDependency) GetDeploymentConfig() (map[string]string, erro
 
 // GetDeploymentServiceEnvVariables returns configuration needed for each service in deployment
 func (r *rdsProductionDependency) GetDeploymentServiceEnvVariables() map[string]string {
-	return map[string]string{}
+	return map[string]string{
+		fmt.Sprintf("%s_ENDPOINT", strings.ToUpper(r.config.Name)): fmt.Sprintf("%s.%s.local", r.config.Config.Rds.DbName, r.appConfig.Name),
```

Comment:
So for postgres this would be something like `POSTGRES_ENDPOINT`? In dev this is currently `POSTGRES`, thoughts on removing the `_ENDPOINT` from the name? Mainly just think it would nice if its consistent between dev and prod

Reply:
Yep, that works.